Platforms/QemuQ35Pkg/QemuQ35Pkg.fdf

```diff
@@ -504,7 +506,9 @@ INF  MmSupervisorPkg/Drivers/StandaloneMmUnblockMem/StandaloneMmUnblockMem.inf
 !endif
 # INF RuleOverride = DRIVER_ACPITABLE SecurityPkg/Tcg/Tcg2Smm/Tcg2Smm.inf
 # COMMENTED OUT DUE TO LACK OF TPM
-# INF  SecurityPkg/Tcg/MemoryOverwriteControl/TcgMor.inf
+INF  SecurityPkg/Tcg/MemoryOverwriteControl/TcgMor.inf
+INF  SecurityPkg/Tcg/MemoryOverwriteRequestControlLock/TcgMorLockSmm.inf
+INF  MdeModulePkg/Universal/Variable/RuntimeDxe/VariableSmm.inf

```

Comment:
`TcgMorLockSmm` and `VariableSmm` are `DXE_SMM_DRIVER` modules. The MM core here is provided by `MmSupervisorCore` which processes `MM_STANDALONE` modules (like `VariableStandaloneMm`). Are you actually seeing them loaded by that core (they should not be)?

Also, they should be in a `!if $(SMM_ENABLED) == TRUE` section.

Reply:
I am seeing these modules loaded, also I notice that MmSupervisorCore is currently included in the FV [if SMM_ENABLED==TRUE](https://github.com/microsoft/mu_tiano_platforms/blob/3fedb0ea0f2cbee8df4de6101e4781cc99191cf0/Platforms/QemuQ35Pkg/QemuQ35Pkg.fdf#L458).

I haven't explored this area of the codebase much. I'll look into this more soon.

Reply:
That makes sense because the MM environment here is Standalone MM (instead of Traditional MM) with the MM Supervisor acting as the MM core.

I suppose you could see `DXE_SMM_DRIVER` modules dispatched by the DXE dispatcher due to their DXE entry point. I'm suggesting in this comment that these Traditional MM (SMM) modules like `VariableSmm` do not get added outside `SMM_ENABLED` blocks and that those blocks currently support Standalone MM instead of Traditional MM.

Reply:
Gotcha :)

Blocked these modules (and the MOR lock test) behind SMM_ENABLED